content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
 lastmod: 2017-02-01
 weight: 50
 ---
```

Comment:
I believe index here like the one at the top of this file (https://raw.githubusercontent.com/jenkins-x/jx-docs/master/content/en/docs/getting-started/setup/boot/clouds/amazon.md) would be helpful.